prqlc/prql-compiler/tests/integration/snapshots/integration__queries__fmt__group_sort_limit_take_1.snap

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
This test seems unnecessary, as we already have `distinct_on.prql`

I'd prefer to be a bit conservative when adding integration tests, since they are slower to execute.

Reply:
[36eb293](https://github.com/PRQL/prql/pull/3988/commits/36eb2931761f275b750cbdc06c2468573a5b115d)

Reply:
> I'd prefer to be a bit conservative when adding integration tests, since they are slower to execute.

Yeah, I think we can be quite generous about unit tests; agree we should try and have fewer, more canonical integration tests if possible

Reply:
I tend to think the opposite. I'm working right now on the MSSQL issue and without integration tests, I feel like it's very very very hard to make sure that
- a dialect is truly supported
- you don't break anything by changing some logic
I don't mean that UT are useless or anything but looking at PRQL scope and its goal, integration tests should be IMHO way more used.
Those integration tests could run only once at night on `master` if we don't want to have them run everytime

Reply:
Yes interesting. 

If they're helpful to you when developing, I would say to use them liberally in that case — we should do the thing that makes making contributions easier.

Personally I prefer to iterate over the very smallest possible thing — so in this case it would be making some SQL queries that work in MSSQL, and then working on getting unit tests to produce those queries. But others prefer a more "integrated" approach of getting PRQL that works on MSSQL in a single loop, even if the loop is slower, and that's OK in moderation imo...